CodeGen/templates/Action.h

```diff
@@ -107,43 +120,53 @@ struct {{data.ModuleAPI}} F{{data.StructName}}GetResultRequest
 };
 
 USTRUCT(Blueprintable)
-struct {{data.ModuleAPI}} F{{data.StructName}}GetResultResponse
+struct {{data.ModuleAPI}} FROS{{data.UEName}}GRRes
 {
 	GENERATED_BODY()
 
 public:
+	{{data.ResultConstantsDef}}
+
 	UPROPERTY()
-	int8 Status = 0;
+	int8 GRResStatus = 0;
 
 	{{data.ResultTypes}}
 
+	FROS{{data.UEName}}GRRes()
+    {
+		{{data.ResultConstructor}}
+    }
```

Comment:
Could it be in initialization list like `FROS{{data.UEName}}GRRes():{{data.ResultConstructor}}`?